doc/deprecated.rst

```diff
@@ -26,6 +26,13 @@ Extensions
 * As of Twig 1.x, the ability to remove an extension is deprecated and the
   ``Twig_Environment::removeExtension()`` method will be removed in 2.0.
 
+* As of Twig 1.x, the ``Twig_ExtensionInterface::initRuntime()`` is deprecated.
```

Comment:
fixed typo
@stof we are not using that anywhere, that should be done in a separate PR.